SDWebImage/SDWebImageTransition.h

```diff
@@ -0,0 +1,45 @@
+/*
+ * This file is part of the SDWebImage package.
+ * (c) Olivier Poitrey <rs@dailymotion.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+#import "SDWebImageCompat.h"
+
+#if SD_UIKIT || SD_MAC
+// Currently it rely on UIKit's `+[UIView transitionWithView:duration:options:animations:completion]` for iOS & tvOS only, for macOS & watchOS it not supported. But just mark this class available for future implementation
```

Comment:
Actually, we can implement this on macOS using the `NSAnimationContext` API. But this need spend more time to do this. Currently I just leave it here :) Maybe I can implement this image transition on macOS if I have extra time